viam-rand-s-mac-main.json

```diff
@@ -0,0 +1 @@
+{"cloud":{"app_address":"https://app.viam.com:443","id":"50dc0a41-0f21-4186-9394-5d26d9e2b053","secret":"4603q9h23gvg93555qnuhi8br2whqg1ntdv9d6qg5215bxxh"}}
```

Comment:
This isn't supposed to be in here, right? Please rotate your secret when you remove this.

Reply:
Yeah, didn't notice it before i pushed, closing this, deleting the branch and rotating the secret in the relocation it's in